### PR TITLE
docs: erpnext italian invoicing (v13)

### PR DESCRIPTION
**Pull request by PawanMeh**
Documentation for PR: https://github.com/frappe/erpnext/pull/19335

### CODE REVIEW

**Comment by PawanMeh** [comment]
@prasadarr kindly review and merge if everything is fine.

File: erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Default Stock UOM should be specified in the Stock Settings Doctype.
+- Enable Check Supplier Invoice Uniqueness in the Accounts Settings Doctype.
+- Create a Zip file with all your supplier invoice XML files.
+
+## Instructions
+
+### On your ERPNext site:
+
+1. Navigate to Import Supplier Invoice doctype and entry the Invoice Series, Company, Supplier Group, Tax Account, Item Code and Default Buying Price list.
+	<img class="screenshot" src="/docs/assets/img/regional/italy/import_einvoice.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]: the top part needs cropping, browser is leaking into the screenshot

File: erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!-- add-breadcrumbs -->
+# Importing e-Invoice from Supplier
+ERPNext has a feature to import supplier invoices from XML files provided by suppliers to the government. Using this you can import Supplier e-invoices into ERPNext. The supplier details like supplier names, address and purchase invoices will get created automatically in the system from the XML files.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]: give some context at the start, what is the govt requirement/what rule is being complied with this feature?

File: erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Prerequisites
+- Default Stock UOM should be specified in the Stock Settings Doctype.
+- Enable Check Supplier Invoice Uniqueness in the Accounts Settings Doctype.
+- Create a Zip file with all your supplier invoice XML files.
+
+## Instructions
+
+### On your ERPNext site:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]:
```suggestion
### On your ERPNext site
```

File: erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md (hunk 2 above)
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]:
```suggestion
As of January 1, 2019, electronic Invoicing is mandatory for domestic businesses operating with domestic B2B and B2C transactions in Italy. ERPNext has a feature to import supplier invoices from XML files provided by suppliers to the government.

ERPNext has a feature to import supplier invoices from XML files provided by suppliers to the government. Using this you can import Supplier e-invoices into ERPNext. The supplier details like supplier names, address and purchase invoices will get created automatically in the system from the XML files.
```

File: erpnext_com/www/docs/user/manual/en/regional/italy/index.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<!-- add-breadcrumbs -->
+# Requirements for Italy
+
+Electronic Invoicing is now obligatory for businesses operating in Italy. ERPNext has a feature to import supplier invoices from XML files provided by suppliers to the government.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/italy/index.md]:
```suggestion
ERPNext aims to support local regulation for different regions in the world. In most cases ERPNext is very flexible so you can easily add Custom Fields and make Custom Reports to support the regulation of your region.

This section contains region specific features for Italy. 
```

File: erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md (hunk 1 above)
Comment (PawanMeh) [reply erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]: Are you asking to crop the image?

Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]: yes

Comment (PawanMeh) [reply erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]: Ok revised screenshot is attached, you can check.

File: erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+2. After entering the above details click on Save.
+
+3. Attach the zip file with XML invoices.
+
+4. Click on Import invoices and the Purchase Invoices will be created. Suppliers would be created if they do not exist in the system already.
+	<img class="screenshot" src="/docs/assets/img/regional/italy/purchase_invoices_created.png">
+
+5. If the file import completes successfully you would see a status of File Import Completed. If there are any errors you can view them from the Error Log.
+	<img class="screenshot" src="/docs/assets/img/regional/italy/file_import_completed.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]: kindly retake screenshot to show top parts, i.e navigation like in the import_einvoice.png screenshot

File: erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+   - Tax Account - The account under which the taxes would be entered for the Purchase Invoices created.
+   - Item Code - The item code which would be used for Purchase Invoice creation.
+   - Default Buying Price list - The default buying price list to be used for the Purchase Invoice.
+
+2. After entering the above details click on Save.
+
+3. Attach the zip file with XML invoices.
+
+4. Click on Import invoices and the Purchase Invoices will be created. Suppliers would be created if they do not exist in the system already.
+	<img class="screenshot" src="/docs/assets/img/regional/italy/purchase_invoices_created.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]: kindly retake screenshot to show top parts, i.e navigation like in the import_einvoice.png screenshot

Comment (PawanMeh) [reply erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md]: Added new images in latest commit

**Pull request merged**